eng/feeds/__snapshots__/azure-core/tspconfig.yaml

```diff
@@ -22,9 +22,9 @@ options:
     model-namespace: false
   "@azure-tools/typespec-ts":
     package-dir: "azure-contoso-rest"
-    generateMetadata: true
-    generateTest: true
-    packageDetails:
+    generate-metadata: true
```

Comment:
i am not sure what is this folder for `eng/feeds`?

Reply:
This is where the feeds for typespec-azure templates reside.